test/integration/fixtures/blocks/core__image__attachment-link.json

```diff
@@ -7,7 +7,11 @@
 			"alt": "",
 			"caption": "",
 			"href": "http://localhost:8888/?attachment_id=7",
-			"linkDestination": "attachment"
+			"linkDestination": "attachment",
+			"behaviors": {
+				"lightbox": true
+			},
+			"lightbox": false
```

Comment:
This is a question for all the tests: Why do we need `behaviors.lightbox = true` and `lightbox = false`?

Reply:
I just touched base with @michalczaplinski on this; there's some inconsistency between this PR and the [behaviors UI PR](https://github.com/WordPress/gutenberg/pull/49972) that he and I will work to resolve between today and early tomorrow, so that should fix this.

Reply:
I've removed it as it's not needed for the Behaviors UI 👍